pkg/plugins/golang/v3/scaffolds/api.go

```diff
@@ -44,9 +44,13 @@ type apiScaffolder struct {
 
 	// force indicates whether to scaffold controller files even if it exists or not
 	force bool
+
+	// useWorkspaces indicates whether to use go.work style workspaces instead of a go module
+	useWorkspaces bool
```

Comment:
Why we would need to inform it every time that we create an API?
I think it would be like multi-group if we change the layout to accept that then, we have the info in the PROJECT file and it would be valid for all apis scaffolded.